ape_etherscan/explorer.py

```diff
@@ -47,23 +52,55 @@ def _client_factory(self) -> ClientFactory:
             )
         )
 
-    def get_contract_type(self, address: AddressType) -> Optional[ContractType]:
+    def get_manifest(self, address: AddressType) -> PackageManifest:
```

Comment:
😎

Reply:
should probably still be optional because it might not be verified

Reply:
fixed